frontend/src/pages/projects/screens/detail/ProjectDetails.stories.tsx

```diff
@@ -0,0 +1,116 @@
+import { mockSecretK8sResource } from '~/__mocks__/mockSecretK8sResource';
+import { mockServingRuntimeK8sResource } from '~/__mocks__/mockServingRuntimeK8sResource';
+import { mockServingRuntimesConfig } from '~/__mocks__/mockServingRuntimesConfig';
+import { mockProjectK8sResource } from '~/__mocks__/mockProjectK8sResource';
+import { mockPVCK8sResource } from '~/__mocks__/mockPVCK8sResource';
+import useDetectUser from '~/utilities/useDetectUser';
+import ProjectDetails from './ProjectDetails';
+
+const handlers = (isEmpty: boolean): RestHandler<MockedRequest<DefaultBodyType>>[] => [
+  rest.get('/api/k8s/api/v1/namespaces/test-project/pods', (req, res, ctx) =>
+    res(ctx.json(mockK8sResourceList(isEmpty ? [] : [mockPodK8sResource({})]))),
+  ),
+  rest.get(
+    '/api/k8s/apis/route.openshift.io/v1/namespaces/test-project/routes/test-notebook',
+    (req, res, ctx) => res(ctx.json(mockRouteK8sResource({}))),
+  ),
+  rest.get('/api/k8s/apis/kubeflow.org/v1/namespaces/test-project/notebooks', (req, res, ctx) =>
+    res(ctx.json(mockK8sResourceList(isEmpty ? [] : [mockNotebookK8sResource({})]))),
+  ),
+  rest.get('/api/k8s/apis/project.openshift.io/v1/projects', (req, res, ctx) =>
+    res(ctx.json(mockK8sResourceList(isEmpty ? [] : [mockProjectK8sResource({})]))),
+  ),
+  rest.get('/api/k8s/api/v1/namespaces/test-project/persistentvolumeclaims', (req, res, ctx) =>
+    res(ctx.json(mockK8sResourceList(isEmpty ? [] : [mockPVCK8sResource({})]))),
+  ),
+  rest.get('/api/k8s/apis/project.openshift.io/v1/projects/test-project', (req, res, ctx) =>
+    res(ctx.json(mockProjectK8sResource({}))),
+  ),
+  rest.get(
+    'api/k8s/apis/serving.kserve.io/v1beta1/namespaces/test-project/inferenceservices',
+    (req, res, ctx) =>
+      res(ctx.json(mockK8sResourceList(isEmpty ? [] : [mockInferenceServiceK8sResource({})]))),
+  ),
+  rest.get('/api/k8s/api/v1/namespaces/test-project/secrets', (req, res, ctx) =>
+    res(ctx.json(mockK8sResourceList(isEmpty ? [] : [mockSecretK8sResource({})]))),
+  ),
+  rest.get(
+    'api/k8s/apis/serving.kserve.io/v1alpha1/namespaces/test-project/servingruntimes',
+    (req, res, ctx) =>
+      res(ctx.json(mockK8sResourceList(isEmpty ? [] : [mockServingRuntimeK8sResource({})]))),
+  ),
+  rest.get(
+    '/api/k8s/apis/serving.kserve.io/v1alpha1/namespaces/test-project/servingruntimes/test-model',
+    (req, res, ctx) => res(ctx.json(mockServingRuntimeK8sResource({}))),
+  ),
+  rest.get('/api/k8s/api/v1/configmaps/servingruntimes-config', (req, res, ctx) =>
+    res(ctx.json(mockServingRuntimesConfig({}))),
+  ),
+];
+
+export default {
+  title: 'ProjectDetails',
+  component: ProjectDetails,
+  parameters: {
+    reactRouter: {
+      routePath: '/projects/:namespace/*',
+      routeParams: { namespace: 'test-project' },
+    },
+    msw: {
+      handlers: handlers(false),
+    },
+  },
+} as ComponentMeta<typeof ProjectDetails>;
```

Comment:
@Gkrumbach07 `as` is very not ideal. We need to consider better templating and direction to avoid this as a requirement for the default export.

Reply:
I believe it is required by storybook for the default export to be that? I might be confused on what you are asking?

Reply:
`as` is never required. As tells TS to override it's thinking. You're trying to solve a type problem by "forcing" TS to be a type. It's not ideal. We should type things with normal types and then export them.

Eg.
```
const baseSettings: TheTypeYouWant = {
   ...
}
export default baseSettings;
```